dprint.json

```
@@ -18,7 +18,10 @@
     "maker-frontend/dist",
     "taker-frontend/dist",
     "**/node_modules",
-    "rocket-cookie-auth/example-http-cookie-auth.sql"
+    "rocket-cookie-auth/example-http-cookie-auth.sql",
+    "**/dist",
+    "**/build",
+    "taker-electron/testnet"
```

```suggestion
    "**/testnet"
```

this will make it easier to ignore all the testnet dirs. 
BTW: what dprint files are in the testnet dir of taker-electron that you wanted to exclude specifically?

---

Also does not exist anymore :D